src/components/users/profile/ko/runtime/profile.html

```diff
@@ -1,4 +1,6 @@
 <div class="panel panel-table">
+    <img
```

Comment:
what is it for?

Reply:
I just added something un-accessible to see if the scan will catch it. I have to delete it :D